lib/mayaHydra/mayaPlugin/viewCommand.cpp

```diff
@@ -41,6 +42,12 @@ constexpr auto _listRenderersLong = "-listRenderers";
 constexpr auto _listActiveRenderers = "-lar";
 constexpr auto _listActiveRenderersLong = "-listActiveRenderers";
 
+constexpr auto _currentProcessMemory = "-cpm";
+constexpr auto _currentProcessMemoryLong = "-currentProcessMemory";
+
+constexpr auto _hdStGPUMem = "-hdm";
+constexpr auto _hdStGPUMemLong = "-hdStGPUMem";
```

Comment:
The "storm" centric-ness in the flag names have been removed in the latest commit. It is now just `hdGPUMem`. 

GetRenderStats() from a renderer's Render Delegate could potentially be any arbitrary data(VtDictionary). As of now, since our tests are mainly with Storm(also the only GPU renderer), the code only returns a valid value in that case else it returns 0. This behavior could be updated in future as we start testing against more offline/gpu renderers.

Hydra has tokens like `HdPerfTokens->gpuMemoryUsed` which we could possibly leverage to construct a better interface. I think it's best we do that when we have additional data points with more renderers